File: 1541.c
#include <stdio.h>
#include <math.h>

int main() 
{

    int a, b, c, aux;
    a = 1;
    do
    {
        
        scanf("%d", &a);
        if (a==0)
        {
          break;  
        }
        scanf("%d%d", &b, &c);
        aux = (a*b*100)/c;
        aux = sqrt(aux);
        printf("%d", aux);

    } while (1);
    
    

    return 0;

}

File: python/1541.py
import math
entra = 0
while True:
    entrada = input().split()
    if entrada == ['0']:
        break
    a, b, c = entrada
    a, b, c = int(a), int(b), int(c)
    aux = math.sqrt((a*b*100)/c)
    aux = int
    print(aux)
